addon/utils/access-panel.js

```diff
@@ -0,0 +1,146 @@
+import { action } from '@ember/object';
+import { addObserver } from '@ember/object/observers';
+import { tracked } from '@glimmer/tracking';
+
+export default class {
+  @tracked displayAccessPanel = false;
+  @tracked displayArchived = false;
+  @tracked searchQuery = '';
+  @tracked contentLoading = false;
+  @tracked accessPanelEntities = [];
+
+  @tracked page = 1;
+  @tracked meta = {};
+
+  constructor(owner, config) {
+    this.owner = owner;
+    this.config = config;
+
+    [
+      `${config.model}.@each.archived`,
+      'displayAccessPanel',
+      'displayArchived',
+      'shouldReloadAccessPanel',
+      'searchQuery'
+    ].forEach((property) => {
+      addObserver(this, property, this.populateAccessPanel);
+    });
+  }
+
+  populateAccessPanel() {
+    if (!this.displayAccessPanel) {
+      return;
+    }
+
+    let model = this.config.model;
```

Comment:
`const model`

Reply:
didn't see it when merging, so i fixed this on master